templates/modcontrol/auditlog.html

```diff
@@ -0,0 +1,46 @@
+$def with ( log, form={} )
```

Comment:
If I recall correctly, other templates don’t have spaces inside the parentheses here.